wgpu/src/api/bind_group_layout.rs

```diff
@@ -18,6 +18,14 @@ pub struct BindGroupLayout {
 #[cfg(send_sync)]
 static_assertions::assert_impl_all!(BindGroupLayout: Send, Sync);
 
+impl BindGroupLayout {
```

Comment:
Any reason that these are sometimes separate impls but otherwise part of existing impls?

Reply:
Not really, I will fix it.